prody/dynamics/adaptive.py

```diff
@@ -375,7 +380,6 @@ def calcAlternatingAdaptiveANM(a, b, n_steps, **kwargs):
         LOGGER.info('\nStarting cycle {0} with {1}'.format(n + 1, getTitle(a, 'structure A')))
         n_modes = calcStep(coordsA, coordsB, n_modes, ensA, defvecs, rmsds, mask=maskA,
                            resetFmin=resetFmin, **kwargs)
-        resetFmin = False
```

Comment:
resetFmin should not be moved down because this is the first iteration and the next one is actually the next iteration